tools/run_desktop_tests.py

```diff
@@ -156,6 +164,9 @@ def run_tests(self, tests_to_run):
 
 
     def log_exec_file(self, filename, result=None):
+        if self.boost_tests:
```

Comment:
Why boost tests are skipped here? Are you planning to log their execution separately?